src/javascript/app/pages/trade/markets.jsx

```diff
@@ -128,9 +128,10 @@ class Markets extends React.Component {
                 symbol: underlying_symbol,
                 name  : this.underlyings[underlying_symbol],
             },
-            markets      : markets_arr,
-            active_market: market_symbol,
-            query        : '',
+            markets             : markets_arr,
+            active_market       : market_symbol,
+            query               : '',
+            openDropdownScrollId: 0,
```

Comment:
it's better to underscore instead of camelCase for variable names: e.g. `open_dropdown_scroll_id`